paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowButton.kt

```diff
@@ -31,47 +50,158 @@ internal fun PaymentMethodRowButton(
     subtitle: String?,
     onClick: () -> Unit,
     contentDescription: String? = null,
-    modifier: Modifier = Modifier,
+    modifier: Modifier = Modifier.fillMaxWidth(),
     trailingContent: (@Composable RowScope.() -> Unit)? = null,
 ) {
-    val contentPaddingValues = if (subtitle != null) {
-        8.dp
-    } else {
-        12.dp
+    Row(
```

Comment:
Why is the row needed? It looks like we can combine the modifiers with the card.